tests/libvmifact/factory.go

```diff
@@ -98,19 +106,27 @@ func NewAlpineWithTestTooling(opts ...libvmi.Option) *kvirtv1.VirtualMachineInst
 }
 
 func NewGuestless(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
-	opts = append(
-		[]libvmi.Option{libvmi.WithResourceMemory(qemuMinimumMemory())},
```

Comment:
So, whenever we want to pass a different memory to the **`NewGuestless`** function. The above code is overriding it with the **`qemuMinimumMemory`** function. So, I have removed it from inside the function and passing it as a function argument.  This way we can avoid this issue, and another thing is that we don't need to explicitly call libvmi.New() in the tests. We can directly call the NewGuestless with the required options and it will take care. 
See here:
https://github.com/kubevirt/kubevirt/pull/12516/files#diff-d9842d3f25d0c190c12e603a60b0af75d5f52f6704866e9db63aeba68bd414f6L1227-R1251

Reply:
Can you point to an example where we use `NewGuestless` with something other than 
 `libvmi.WithResourceMemory(libvmifact.QemuMinimumMemory())`? The above link isn't working for me for some reason. Again given your changes to `QemuMinimumMemory` this seems pointless.

Reply:
can you check this once?
https://github.com/kubevirt/kubevirt/blob/6ed1da062e8765b857fe434fd74bedd81f8d3a64/tests/vmi_lifecycle_test.go#L240

Reply:
Ah sorry github hid the diff for that file!